app/javascript/dashboard/components/layout/Sidebar.vue

```diff
@@ -90,11 +90,61 @@
           </label>
         </a>
       </div>
+      <div class="modal-footer delete-item">
```

Comment:
Why is named `delete-item`?

Reply:
reused this. It was already being used in DeleteModal.
Let me know, if I need to change this.